src/API/Notes.php

```diff
@@ -415,6 +415,29 @@ public function prepare_query_for_response( $query ) {
 		return admin_url( $query );
 	}
 
+	/**
+	 * Maybe add a nonce to a URL.
+	 *
+	 * @link https://codex.wordpress.org/WordPress_Nonces
+	 *
+	 * @param string $url The URL needing a nonce.
+	 * @param string $action The nonce action.
+	 * @param string $name The nonce anme.
+	 * @return string A fully formed URL.
+	 */
+	private function maybe_add_nonce_to_url( string $url, string $action = '', string $name = '' ) : string {
+		if ( empty( $action ) ) {
+			return $url;
+		}
+
+		if ( empty( $name ) ) {
+			// Allow wp_nonce_url to use a default paramater name.
```

Comment:
Just need to update the comment to replace `wp_nonce_url` as it is not used anymore.

Reply:
Thanks! fixed in 225c187.